homework_1/micrograd/test_nn.ipynb

```diff
@@ -0,0 +1,685 @@
+{
```

Comment:
<p>what about <code>optimizer.zero_grad</code> in <code>OptimizerCallback</code> ? Should be done <code>on_batch_start</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-team/dl-course/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>also let's change signature of all callbacks methods to (<code>self, runner: "Runner"</code>) <span class="ql-cursor">﻿</span></p><p> </p>